### ISSUE TITLE
Please upgrade to coffeelint version `>=2`

### ISSUE THREAD

**Issue by cueedee** [vojtajina/grunt-coffeelint#69]
Now that coffeelint has bumped its major version, it will no longer match the semver range of `^1` in your [`dependencies`](https://github.com/vojtajina/grunt-coffeelint/blob/v0.0.16/package.json#L41).

**Comment by mwittig** [comment]
Ok, will do. This will, however, will no longer work with node v4 as v6.9 is the minimum required version.

**Comment by mwittig** [comment]
Created a separate branch for integration: coffeelint2-dev

**Comment by cueedee** [comment]
@mwittig, can it be assumed that this change, when it is released will result in a major version bump?

**Comment by mwittig** [comment]
@cueedee Yes, I think it has to be a major version bump for various reasons
* dependency on node v6.9 is a breaking change in my opinoin as some users may still need to work on legacy projects
* coffeescript 2 introduces a few breaking changes which might also support the linting process. I haven't looked into the details yet, however - see http://coffeescript.org/#breaking-changes

**Comment by cueedee** [comment]
Hi Again, could you provide any insight into when you plan to release? (But before you do, please note my comment on commit 4fe4f73)

**Comment by cueedee** [comment]
_testing, testing,.. is this thing on? @mwittig?_
